src/main/java/org/jabref/model/entry/identifier/RfcId.java

```diff
@@ -0,0 +1,79 @@
+package org.jabref.model.entry.identifier;
+
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RfcId implements Identifier {
+
+    private static final Pattern RFC_PATTERN = Pattern.compile("^rfc\\d+$", Pattern.CASE_INSENSITIVE);
+    private static final Pattern RFC_URL_PATTERN = Pattern.compile("rfc(\\d+)", Pattern.CASE_INSENSITIVE);
+
+    private final String rfcString;
+
+    public RfcId(String rfcString) {
+        this.rfcString = Objects.requireNonNull(rfcString).trim().toLowerCase();
+    }
+
+    public static Optional<RfcId> parse(String input) {
+        Matcher rfcMatcher = RFC_PATTERN.matcher(input);
+        if (rfcMatcher.matches()) {
+            return Optional.of(new RfcId(input));
+        }
+
+        Matcher urlMatcher = RFC_URL_PATTERN.matcher(input);
+        if (urlMatcher.find()) {
+            String parsedRfcId = "rfc" + urlMatcher.group(1);
+            return Optional.of(new RfcId(parsedRfcId));
+        }
+
+        return Optional.empty();
+    }
+
+    public boolean isValid() {
```

Comment:
Hi, maybe I understand something wrong, so I don't need to match plain in this part, but only need to match URLs in this format("https://www.rfc-editor.org/rfc/" + rfc7276.html#section-2.2.2)?

Reply:
Can the modified part meet your requirements?

Reply:
> Hi, maybe I understand something wrong, so I don't need to match plain in this part, but only need to match URLs in this format("https://www.rfc-editor.org/rfc/" + rfc7276.html#section-2.2.2)?

You do nlt match the URL at all there. therefore I wondered.

Reply:
Because I was worried that there might be other formats of URLs, I considered just matching the format of rfc****. 
My mistake. I am really sorry for causing you trouble.

Reply:
The method is nowhere used. REmove.

If you would implement it correctly, you need to have to check both the URL and just the ID.